lib/rubocop/cop/mixin/autocorrect_alignment.rb

```diff
@@ -10,6 +10,14 @@ def configured_indentation_width
         config.for_cop('IndentationWidth')['Width']
       end
 
+      def indentation(node)
+        offset(node) + (' ' * configured_indentation_width)
```

Comment:
I'm guessing where allocating this ` ` a lot. He should probably make it a frozen constant.